src/SupervisionSuccess/components/ChangeInRevocations/ChangeInRevocations.jsx

```diff
@@ -5,18 +5,23 @@ import debounce from "debounce";
 
 import "./ChangeInRevocations.css";
 
-const ChangeInRevocations = ({ changeInRevocations, onChangeInRevocationsChange }) => {
+const ChangeInRevocations = ({
+  state,
+  finalPopulation,
+  changeInRevocations,
+  onChangeInRevocationsChange,
+}) => {
   const renderThumb = useCallback(
-    (props, state) => (
+    (props, thumbState) => (
       <span {...props} className="revocations-slider_thumb">
-        <span className="revocations-slider_percent">{state.valueNow}%</span>
+        <span className="revocations-slider_percent">{thumbState.valueNow}%</span>
         <span className="revocations-slider_hint">
-          <b className="revocations-slider_hint-count">6324</b> Violations resulting in State
-          incarceration
+          <b className="revocations-slider_hint-count">{finalPopulation}</b> Violations resulting in{" "}
```

Comment:
I think `finalPopulation` is not the right number. It looks like this is the `total population` column (D) in the Model tabs of the spreadsheet. What we really want this to be is the `revocations` column (C) in the same tabs of the spreadsheet. "Violations resulting in incarceration" is, somewhat imprecisely, referring to the number of revocations here.